src/metabase/email/pulse.mustache

```diff
@@ -129,7 +137,9 @@
                 </tr>
               {{/filters}}
             </table>
-            <hr style="border-width: 0; background: #F0F0F0; height: 1px; margin-top: 20px; margin-bottom: 20px;">
+            {{^dashboardHasTabs}}
```

Comment:
as noted in the design, we only want this initial `hr` for dashboard with no tabs